src/com/teamcautionrobotics/robot2018/Climb.java

```diff
@@ -0,0 +1,25 @@
+package com.teamcautionrobotics.robot2018;
+
+import edu.wpi.first.wpilibj.VictorSP;
+
+public class Climb {
+
+    private VictorSP climbMotor;
+
+    public Climb(int motorPort) {
+        climbMotor = new VictorSP(motorPort);
+    }
+
+    public void move(double power) {
```

Comment:
Could you add JavaDoc for the `Climb.move()` and `Intake.run()` functions. The other functions probably do not need it.